fedimint-core/src/module/mod.rs

```diff
@@ -578,6 +580,26 @@ pub trait CommonModuleInit: Debug + Sized {
     fn decoder() -> Decoder;
 }
 
+/// Can be used to send a notification to consensus server,
+/// to trigger immediate [`ServerModule::consensus_proposal`] call.
+///
+/// Useful for modules to optionally decrease latency of consensus
+/// item handling.
+#[derive(Debug, Clone)]
+pub struct ConsensusProposalTrigger(watch::Sender<u64>);
```

Comment:
can this be `watch::Sender<()>`, Sender and Receiver don't require T: PartialEq so it can't check if value in `Receiver::changed()`.

Reply:
Maybe but I'm spooked about the "changed" aspect of the unit so whatever. 🤣